backend/controllers/createDpsUser.js

```diff
@@ -88,6 +87,10 @@ const createDpsUserFactory = (getCaseloads, createDpsUser, createUserUrl, manage
   return { index, post }
 }
 
+function removeForwardApostrophe(str) {
```

Comment:
This would probably sit neater in utils.js

Reply:
Done